sections/what_is_rust.tex

```diff
@@ -8,32 +8,53 @@ \section{What is Rust?}
 		that runs blazingly fast, prevents nearly all segfaults,\\
 		and guarantees thread safety.»\\
 		\vspace{0.5em}
-		{\normalfont \small --- www.rust-lang.org}
+		{\normalfont \small --- www.rust-lang.org (until 2018)}
+	\end{quote}
+
+	\vspace{2em}
+
+	\begin{quote}
+		«A language empowering everyone\\
+		to build reliable and efficient software.»\\
+		\vspace{0.5em}
+		{\normalfont \small --- www.rust-lang.org (since 2019)}
```

Comment:
So what was it between 2018 and 2019?

Reply:
Since and until are both meant inclusively, but I can change the time specification above to "until 2019" if you prefer inclusive-since-but-exclusive-until :wink:

Reply:
Ah of course, well classic off by one error :wink: So in Rust terms you meant `..=2018` :stuck_out_tongue_closed_eyes:

Reply:
I found it confusing and would prefer "until 2019". But feel free to just resolve.

Reply:
Updated